README.md

```diff
@@ -381,12 +381,106 @@ rer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" https://metrics.
 
 ### Writing Release Notes
 
-Release notes are maintained in the [change log](CHANGELOG.md) and follow guidelines
-based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). We include additional
-guidelines for increased readability and consistency:
-
-* Changes should be grouped by section, either as an enhancement, bug fix, or
-  internal change
-* References to code or configuration should render as literals
-* Bug fixes should link to bug reports (GitHub Issues or Jira items)
-* Features or enhancements should link to formal RFEs (GitHub Issues or Jira items)
+Release notes are maintained in the [changelog](CHANGELOG.md) and follow
+guidelines based on [keep a changelog](https://keepachangelog.com/en/1.0.0/).
+This section describes additional guidelines, conventions, and the overall
+process for writing release notes.
+
+#### Guidelines
+
+* Each release should contain release notes
+* Changes should be applicable to at least one of the six types listed below
+* Use literals for code and configuration
+  - ``defaultScanSettingsSchedule``
+  - ``nodeSelector``
```

Comment:
These literal examples could be collapsed into the previous bullet and calling out an example, otherwise it's not really clear what these code snippets are doing here.

  (examples: ``defaultScanSettingsSchedule`` or ``nodeSelector``)